src/lib.rs

```diff
@@ -1,4 +1,8 @@
 #![doc = include_str!("../README.md")]
+
+pub mod utils;
```

Comment:
The module above the import statements is a bit weird formatting wise, add a line break or two and put it below the imports (that's more idiomatic)